test/runner/lib/cloud/vcenter.py

```diff
@@ -149,13 +248,30 @@ def get_environment_config(self):
         """
         :rtype: CloudEnvironmentConfig
         """
-        env_vars = dict(
-            VCENTER_HOST=self._get_cloud_config('vcenter_host'),
-        )
+        vmware_test_platform = os.environ.get('VMWARE_TEST_PLATFORM', '')
```

Comment:
Environment variables won't be available when delegation such as `--docker` is used. To guarantee settings are available after delegation, use `_set_cloud_config` to preserve them. Example:

https://github.com/ansible/ansible/blob/690037876648e515fdf1e9d6dd82c760b9bd0f72/test/runner/lib/cloud/vcenter.py#L135

To retrieve values use `_get_cloud_config`. Example:

https://github.com/ansible/ansible/blob/690037876648e515fdf1e9d6dd82c760b9bd0f72/test/runner/lib/cloud/vcenter.py#L153